tools/Microsoft.Health.Extensions.BuildTimeCodeGenerator/Sql/CreateOrAlterProcedureVisitor.cs

```diff
@@ -248,6 +255,50 @@ private MemberDeclarationSyntax AddPopulateCommandMethodForTableValuedParameters
                                     .WithNameColon(NameColon(ParameterNameForParameter(p)))).ToArray())));
     }
 
+    private MethodDeclarationSyntax[] AddGetOutputMethod(CreateOrAlterProcedureStatement node)
+    {
+        List<TypeSyntax> outputTypes = new List<TypeSyntax>();
+        List<ExpressionSyntax> outputExpressions = new List<ExpressionSyntax>();
+        IEnumerable<ProcedureParameter> procedureParameters = node.Parameters.Where(p => p.Modifier == ParameterModifier.Output);
```

Comment:
nit: Let's call `ToList()` here so you can easily call `.Count` below

Reply:
Thanks, updated.